Polygon_mesh_processing/doc/Polygon_mesh_processing/NamedParameters.txt

```diff
@@ -152,6 +152,30 @@ the property map containing information about edges of the input polygon mesh be
 a default property map where no edge is constrained is provided.
 \cgalNPEnd
 
+\cgalNPBegin{ maximum_number_of_patches } \anchor PMP_maximum_number_of_patches
+contains the surface patch beeing treated +1.\n
+\b Type : `std::size_t`* \n
+\b Default : a pointer to a `std::size_t` with value 1.
+\cgalNPEnd
+
+\cgalNPBegin{ halfedge_is_feature_map } \anchor PMP_halfedge_is_feature_map
```

Comment:
Implementation-wise, it makes sense to have halfedges, because the info is stored in the halfedges (for `Polyhedron_3`), but API-wise, I also think that it is edges.
With halfedges, the API is open to inconsistencies where opposite halfedges disagree on their "feature or not" status